types/module_contract_name.go

```diff
@@ -53,6 +53,25 @@ func (cn ModuleContractName) String() string {
 }
 
 func (cn ModuleContractName) Match(other ModuleContractName) bool {
```

Comment:
```suggestion
// Match performs a match of one module contract to another
// Typically, a connection or module (`other`) is validated to follow the contract (`cn`)
func (cn ModuleContractName) Match(other ModuleContractName) bool {
```

I always forget which operand does what. Here is a suggestion for text to help clarify the algorithm.